notebooks/data-sources/oc-github-repo/github_PR_EDA.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>:+1: you can remove the comment as I assume it's to explain to the original developer (me) why you made this change.</p><p> </p><p>Do you want time to merge to be float or Int? the other values  here are Int and I think you'll have *.0 for all floats.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/287/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Yep, the original developer as well as any future developers who might work on this :)</p><p> </p><p>We wanted to make it int but because it contains nan's, pandas won't allow that conversion, so we had to make it float</p>